hci/event/events/vendor_specific/gap_device_discovery_done.py

```diff
@@ -24,8 +24,9 @@ def address_type(self):
         @property
         def address(self):
             OFFSET, SIZE_OCTETS = 2, 6
-            address = self._data[OFFSET:OFFSET + SIZE_OCTETS]
-            return address
```

Comment:
`return address[::-1]` is enough.

Reply:
Actually, it is misleading, since the variable `address` is the inverted address. That way, it seems you are returning the inverted address.
So 3 solutions are possible. Either call `address` -> `data`, which would be generic, or call `address` -> `inverted_address`, or simply invert (`[::-1]`) while slicing, as in `address = self._data[OFFSET:OFFSET + SIZE_OCTETS][::-1]`